composable/useClickOutside.ts

```diff
@@ -0,0 +1,30 @@
+export const useClickOutside = (
+  elTargetRef: any,
+  // elTargetRef: HTMLElement | null,
```

Comment:
only with  `elTargetRef: any` I have no ts errors, with the `elTargetRef: HTMLElement | null` - I get ts errors, even though the argument it takes is const root of that ts type defined in SortFileList - 19 line.